app/assets/javascripts/utilities/buildArticleHTML.js

```diff
@@ -278,7 +278,7 @@ function buildArticleHTML(article) {
     }
 
     var saveButton = '';
-    if (article.class_name === 'Article') {
+    if (article.class_name === 'Article' && article.user_id != currentUserId) {
```

Comment:
```suggestion
    if (article.class_name === 'Article' && article.user_id !== currentUserId) {
```
ah, Javascript 😄

Reply:
TIL `!==`

Reply:
@aitchiss I'm glad you brought this up!

So I did have it as `!==` BUT figured out that there were failures because `article-user_id` is an Integer, but `currentUserId` is a string.

Should I change `currentUserId` to an integer, so that the stricter `!==` doesn't invalidate accurate comparisons?

Reply:
Oh interesting! I guess in that case let's keep it as you have it, but maybe we can throw in a comment about this (to make sure it doesn't accidentally get refactored in future!)